{{cookiecutter.project_name}}/Makefile

```diff
@@ -24,15 +24,32 @@ for line in sys.stdin:
 endef
 export PRINT_HELP_PYSCRIPT
 
-BROWSER := python -c "$$BROWSER_PYSCRIPT"
+# Mandatory tools
+PIP := $(shell command -v pip 2> /dev/null)
+PYTHON := $(shell command -v python 2> /dev/null)
+BROWSER := $(ENVPIP) -c "$$BROWSER_PYSCRIPT"
```

Comment:
Looks weird to use a variable here that is defined below. I guess it works because the variable is evaluated during runtime.

Reply:
you are totally right. Also it should be ENVPYTHON